README.rst

```diff
@@ -43,6 +43,37 @@ __ `OpenBSD package`_
 __ `openSUSE package`_
 __ `Ubuntu package`_
 
+Docker
+------
+
+A docker package exists for this project: 
+
+  $ git pull ghcr.io/rss2email/rss2email:master
+
+docker-compose.yaml
+~~~~~~~~~~~~~~~~~~~
+  version: '2'
+  services:
+    rss2email:
+      image: ghcr.io/rss2email/rss2email:master
+      volumes:
+        - ./config:/config
+        - ./data:/data
+      entrypoint: "/venv/bin/python3 -c 'import threading; threading.Event().wait()'"
+      labels:
+        chadburn.enabled: "true"
+        chadburn.job-exec.rss2email.schedule: "@every 5m"
```

Comment:
We shouldn't be suggesting users run r2e more than once per day.

Reply:
agreed, I will bring this in line with cron example